temporalcli/commands.workflow_test.go

```diff
@@ -308,6 +324,7 @@ func (s *SharedServerSuite) testTerminateBatchWorkflow(json bool) *CommandResult
 	// Send batch terminate with a "y" for non-json or "--yes" for json
 	args := []string{
 		"workflow", "terminate",
+		"--rps", fmt.Sprint(rps),
```

Comment:
This introduces a delay in the test suite of up to 1s. Not sure how to avoid that while still testing the Server end-to-end.

Reply:
We may have to accept some things cannot reasonably tested in CLI without slowing all tests down.